README.md

```diff
@@ -17,7 +17,17 @@ const updatedCode = updater('module.exports = { /* ... */ };', 'xunit');
 updateCode === `module.eports = { reporter: "xunit" };`
 ```
 
+```js
+const updater = require('babel-update-ember-cli-test-reporter');
+
+const updatedCode = updater('module.exports = { /* ... */ };', 'xunit', { report_file: 'test_results.xml' });
```

Comment:
I would if we should simply update the API to be, thoughts?

```js
updater(code, { /* ... properties ... */ });
```
Which would become:

```js
updater(code, { reporter: 'xunit' });
updater(code, { reporter: 'xunit', report_file: "build/test_results.xml" });
```

Since we have no consumers we can easily change it without consequences.

Reply:
Good point. Let me adjust

Reply:
:shipit: